src/App.js

```diff
@@ -75,6 +75,8 @@ export default class App extends React.Component {
             changeAddress: undefined,
             rewardAddress: undefined,
             usedAddress: undefined,
+            usedAddresses: undefined,
```

Comment:
Why are you keeping `usedAddress`? Its not a method of CIP-30 and is just duplication

Reply:
It is not hard to get it removed, but this specific field is already ingrained in our test automation suite as we use this dapp extensively for our testing purposes. I talked with Lukasz and we came to conclusion that this is not a big deal, since this is only a test dapp used for our testing purposes. 
If you think differently, let me know and I can try to find some time to do some refactoring in our tests to accommodate this change in dapp.